### PR TITLE
fix(core): use patched concrete-fftw to avoid fftw generic simd256 bug

### PR DESCRIPTION
**Pull request by IceTDrinker**
### Resolves:

closes https://github.com/zama-ai/concrete-core-internal/issues/295

FFTW bug report : https://github.com/FFTW/fftw3/issues/294

Credits to @sarah-ek for finding the bug in the first place and the repro code

### Checklist 

(Use '[x]' to check the checkboxes, or submit the PR and then click the checkboxes)

* [ ] Tests for the changes have been added (for bug fixes / features)
* [ ] Docs have been added / updated (for bug fixes / features)
* [ ] The PR description links to the related issue (to link an issue, use '#XXX'.)
* [ ] The tests on AWS have been launched and are successful (comment with @slab-ci cpu_test and/or @slab-ci gpu_test to trigger the tests)
* [ ] The draft release description has been updated
* [ ] Check for breaking changes (including serialization changes) and add them to commit message following the conventional commit [specification][conventional-breaking]

[conventional-breaking]: https://www.conventionalcommits.org/en/v1.0.0/#commit-message-with-description-and-breaking-change-footer

### CODE REVIEW

**Comment by IceTDrinker** [comment]
@slab-ci cpu_test

**Comment by IceTDrinker** [comment]
@slab-ci cpu_test

**Comment by IceTDrinker** [comment]
@slab-ci gpu_test

**Comment by IceTDrinker** [comment]
@slab-ci gpu_test

**Comment by IceTDrinker** [comment]
@slab-ci cpu_test

File: concrete-core/Cargo.toml [hunk 1]
```diff
@@ -87,10 +86,7 @@ backend_cuda = ["concrete-cuda"]
 __profiling = []
 __private_docs = []
 __commons_parallel = ["rayon", "concrete-csprng/parallel"]
-__commons_serialization = [
-    "serde",
-    "serde/derive",
-]
+__commons_serialization = ["serde", "serde/derive"]
```
Comment (IceTDrinker) [inline concrete-core/Cargo.toml:89]: this is just my formatter doing this :| we probably need a toml formatter in our tools to stop having these diffs

**Review by aPere3 — APPROVED** [review]

**Pull request merged**